src/language/utils.ts

```diff
@@ -74,7 +74,7 @@ export function isBlockScopedVariable(
     node: ts.VariableDeclaration | ts.VariableStatement,
 ): boolean {
     if (node.kind === ts.SyntaxKind.VariableDeclaration) {
-        const parent = node.parent!;
```

Comment:
A lot of the changes are due to `node.parent` becoming required now. Therefore the non-null assertions are unnecessary.
https://github.com/Microsoft/TypeScript/blob/v3.1.6/lib/typescript.d.ts#L480